### ISSUE TITLE
Support AutoSQL

### ISSUE THREAD

**Issue by superbobry** [superbobry/big#7]
> BigBed files can store extra fields in addition to the predefined BED fields. If you add extra fields to your bigBed file, you must include a .as (AutoSQL) format file describing the fields.

Motivation and details on AutoSQL are described [here](http://www.linuxjournal.com/article/5949). At the moment we've decided to restrict `big` to [predefined](https://genome.ucsc.edu/FAQ/FAQformat.html#format1) BED fields, but we'll have to deal with AutoSQL at some point.

**Comment by iromeo** [comment]
`2017-11-09 17:35:29] BigFile [WARN] AutoSQL queries are unsupported`
Logging is too aggressive here, e.g. it spoils `epigenome` project  log file because `epigenome` browser opens big file on each rendering event.

I think it is better to log it as "trace" info or even switch warning off.

**Comment by superbobry** [comment]
Yep, just switch it off. This AutoSQL thing is a mess, I doubt we would ever want to implement it.
